src/routing.ts

```diff
@@ -54,7 +54,7 @@ export const routingCycle = ({
       });
     } else if (element instanceof ServeStatic) {
       if (staticCb) {
-        staticCb(path, _serveStatic(...element.params));
```

Comment:
can we move this functionality to the class method `ServeStatic::apply()` is similar?